.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -21,6 +21,7 @@ _Please go through this checklist and make sure all applicable tasks have been d
 
 - [ ] Write tests
 - [ ] Make sure all tests pass
+- [ ] Review performance test results. Ensure that any degradations to performance numbers are understood and justified.
```

Comment:
@franvila what would you think to this?

Reply:
Wow! Thanks! I cannot image a better place to have it